custom/icds_reports/tests/agg_tests/__init__.py

```diff
@@ -27,7 +27,7 @@
 OUTPUT_PATH = os.path.join(os.path.dirname(__file__), 'outputs')
 
 
-@timelimit(480)
+@timelimit(48000000)
```

Comment:
This seems a bit excessive. Maybe we should double it to 960? 480 is already about twice as long as it takes on travis. Did something change to make this much longer?